src/main/scala/org/apache/mesos/chronos/scheduler/jobs/Containers.scala

```diff
@@ -16,17 +16,37 @@ object NetworkMode extends Enumeration {
   val BRIDGE, HOST = Value
```

Comment:
Should also add `USER` here: https://github.com/mesos/chronos/blob/master/src/main/resources/ui/models/JobForm.js#L217-L221

I just noticed a copy-paste artefact after that `return` statement too.